### ISSUE TITLE
Clarify minOmega and maxOmega in geometry yaml

### ISSUE THREAD

**Issue by johnjasa** [WISDEM/WISDEM#241]
## Description
In some geometry yamls we have `minOmega` and `maxOmega` values, which appear to be holdovers from previous versions of WISDEM (in [this example](https://github.com/WISDEM/WISDEM/blob/develop/examples/02_reference_turbines/IEA-3p4-130-RWT.yaml#L653), for instance).
These values do not actually influence the internal `minOmega` and `maxOmega` values within the OpenMDAO problem, and instead `VS_minspd` and `VS_maxspd` set the corresponding omega values, as shown in [`gc_WT_InitModel.py`](https://github.com/WISDEM/WISDEM/blob/develop/wisdem/glue_code/gc_WT_InitModel.py#L1081).

The existence of `minOmega` and `maxOmega` has confused a user who wanted to change those values, found them within the OpenMDAO problem, then set them in the yaml, but did not see the corresponding changes.

I see a few options forward here:
1. Remove `minOmega` and `maxOmega` from any examples to avoid confusion.
2. Change `gc_WT_InitModel.py` to use the min and max omegas as set in the geometry yaml instead of the `VS_minspd` and max speed values.

I think I prefer option 2. for clarity, but would like to get input before introducing a backwards-incompatible change to the input yamls. What do you all think?

**Comment by gbarter** [comment]
I agree that min/maxOmega is more intuitive, but I think VS_min/maxspd is more consistent with other controller knobs appearing in WEIS.  I think @ptrbortolotti has been the most deeply engaged with these settings.  Either way, you are right that if we are not consistent then we will be in trouble.

**Comment by ptrbortolotti** [comment]
I'm sorry, I forgot about this issue. Garrett is right, minOmega and maxOmega are outdated and I've just removed them from the yamls in the branch analysis_opts_blade, which will be pulled to develop tomorrow. The user still has the option to apply minimum and maximum allowable rotor speed with the inputs control->torque->inputs VS_minspd and control->torque->VS_maxspd and a maximum allowable tip speed with the input control->supervisory->maxTS
I hope this helps clear the fog about this
